recipes/list_items/contact_info/contact_info.vue

```diff
@@ -29,48 +29,41 @@
               :avatar-class="['d-ba d-baw4 d-bc-white d-bar-pill', { 'd-mln24': index > 0 }]"
             >
               <img
+                v-if="avatar.src"
                 data-qa="dt-contact-avatar"
                 :src="avatar.src"
                 :alt="avatar.initials"
               >
-            </dt-avatar>
-            <dt-avatar
-              v-else-if="avatar.initials"
-              :size="avatarSize"
-              :seed="avatar.seed"
-              :initials="avatar.initials"
-              :overlay-icon="avatar.icon"
-              :overlay-text="avatar.text"
-              overlay-class="d-mn4 d-ba d-baw4 d-bc-white d-box-unset"
-              :avatar-class="['d-ba d-baw4 d-bc-white d-bar-pill', { 'd-mln24': index > 0 }]"
-            >
-              {{ avatar.initials }}
+              <div v-else-if="avatar.initials">
+                {{ avatar.initials }}
+              </div>
+              <div v-else>
+                <dt-icon :name="avatarIcon" />
+              </div>
```

Comment:
Don't need to wrap the icon on a `div` that'd prevent the avatar to correctly define the slot content type